modules/ove-about.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * overview/ove-overview.adoc
+
+:_module-type: CONCEPT
+[id="ove-about_{context}"]
+= About {product-title}
+
+While all editions of Red Hat OpenShift include link:https://www.redhat.com/en/technologies/cloud-computing/openshift/virtualization[{virt}], {product-title} is a cost-effective, virtualization-only option that does not include container-based or cloud-native features for applications. If you migrate from a traditional virtualization platform to {product-title}, you can add containerization and modernization features later by switching to a different edition of OpenShift.
```

Comment:
Is OVE only compatible with OCP? If so, please update your OpenShift usage.

Based on the [product name list](https://docs.google.com/spreadsheets/u/2/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/pubhtml?gid=1259317633&single=true), if "Red Hat Openshift" is correct, you always use the whole phrase: 
>Do not use "OpenShift" alone to refer to the Red Hat product family or a specific product. Only use to refer to the technology in general.

Reply:
Changing instances of "OpenShift" (outside of direct references to article titles) to Red Hat OpenShift.